docs/src/main/asciidoc/quartz.adoc

```diff
@@ -0,0 +1,331 @@
+3. The create a new `Task` with the current start time.
+4. Persist the task in database using https://quarkus.io/guides/hibernate-orm-panache[Panache].
+
+== Updating the application configuration file
+
+Edit the `application.properties` file and add the below configuration:
+[source,shell]
+----
+# Quartz configuration
+quarkus.quartz.clustered=true <1>
+quarkus.quartz.store-type=db <2>
+
+# Datasource configuration.
+quarkus.datasource.url=jdbc:postgresql://localhost/quarkus_test
+quarkus.datasource.driver=org.postgresql.Driver
+quarkus.datasource.username=quarkus_test
+quarkus.datasource.password=quarkus_test
+
+# Hibernate configuration
+quarkus.hibernate-orm.database.generation=none
+quarkus.hibernate-orm.log.sql=true
+quarkus.hibernate-orm.sql-load-script=no-file
+
+# flyway configuration
+quarkus.flyway.connect-retries=10
+quarkus.flyway.table=flyway_quarkus_history
+quarkus.flyway.migrate-at-start=true
+quarkus.flyway.baseline-on-migrate=true
+quarkus.flyway.baseline-version=1.0
+quarkus.flyway.baseline-description=Quartz
+----
+
+1. Indicate that the scheduler will be run in clustered mode
+2. Use the database store to persists job related information so that they can be shared between nodes
+
+== Updating the resource and the test
+
+Edit the `TaskResource` class, and update the content to:
+
+[source,java]
+----
+package org.acme.quartz;
+
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/tasks")
+@Produces(MediaType.APPLICATION_JSON)
+public class TaskResource {
+
+    @GET
+    public List<Task> listAll() {
+        return Task.listAll(); <1>
+    }
+}
+----
+1. Retrieve the list of created tasks from database
+
+We also need to update the tests. Edit the `TaskResourceTest` class to match:
+
+[source, java]
+----
+package org.acme.quartz;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+public class TaskResourceTest {
+
+    @Test
+    public void tasks() throws InterruptedException {
+        Thread.sleep(1000); // wait at least a seconds to have the first task created
+        given()
+                .when().get("/tasks")
+                .then()
+                .statusCode(200)
+                .body("size()", is(greaterThanOrEqualTo(1))); <1>
+    }
+}
+----
+1. Ensure that we have a `200` response and at least one task created
+
+== Creating Quartz Tables
+Add a SQL migration file named `src/main/resources/db/migration/V2.0.0__QuarkusQuartzTasks.sql` with the content copied from
+file with the content from this {quickstarts-blob-url}/quartz-quickstart/src/main/resources/db/migration/V2.0.0__QuarkusQuartzTasks.sql[Quartz Tasks migrationfile].
```

Comment:
`with the content copied from file with the content from this`?

Reply:
trailing `this`, we can drop it.